xbmc/guilib/GUIFontTTF.h

```diff
@@ -43,6 +47,7 @@ typedef struct FT_StrokerRec_ *FT_Stroker;
 
 typedef uint32_t character_t;
 typedef std::vector<character_t> vecText;
+typedef std::vector<hb_glyph_info_t> vecGlyphInfo;
```

Comment:
I would avoid the typedef completely as it's not a complex type (it's just a vector)

Reply:
kept for consistency with the other variables defined above

Reply:
done